### ISSUE TITLE
Can't compile with python

### ISSUE THREAD

**Issue by Corna** [xournalpp/xournalpp#110]
I can't compile with python enabled:

```
plugin/python/bindings/PyXournal.cpp: In function ‘PyObject* PyXournal_getCurrentPageBackground(PyXournal*)’:
plugin/python/bindings/PyXournal.cpp:491:29: error: ‘class PageRef’ has no member named ‘getBackgroundType’
  return PyInt_FromLong(page.getBackgroundType());
                             ^
plugin/python/bindings/PyXournal.cpp: In function ‘PyObject* PyXournal_setCurrentPageBackground(PyXournal*, PyObject*)’:
plugin/python/bindings/PyXournal.cpp:536:7: error: ‘class PageRef’ has no member named ‘setBackgroundType’
  page.setBackgroundType((BackgroundType)backgroundType);
       ^
```

I've noticed that PageRef is a pointer to a XojPage, which has a get(set)BackgroundType method
I've fixed it by simply replacing the '.' with an '->' in lines 491 and 536 of src/plugin/python/bindings/PyXournal.cpp
Is it correct? Can you commit this change? Here's the diff

```
diff --git a/src/plugin/python/bindings/PyXournal.cpp b/src/plugin/python/bindings/PyXournal.cpp
index d4c94ac..289da2e 100644
--- a/src/plugin/python/bindings/PyXournal.cpp
+++ b/src/plugin/python/bindings/PyXournal.cpp
@@ -488,7 +488,7 @@ PyXournal_getCurrentPageBackground(PyXournal* self)
                return NULL;
        }
 
-       return PyInt_FromLong(page.getBackgroundType());
+       return PyInt_FromLong(page->getBackgroundType());
 }
 
 static PyObject*
@@ -533,7 +533,7 @@ PyXournal_setCurrentPageBackground(PyXournal* self, PyObject* args)
                return NULL;
        }
 
-       page.setBackgroundType((BackgroundType)backgroundType);
+       page->setBackgroundType((BackgroundType)backgroundType);
 
        Py_RETURN_NONE;
 }
```

I've also had to manually create two links to make it build

    /include/python -> /usr/include/python2.7
    /lib/python/config/python.so -> /usr/lib/python2.7/config-x86_64-linux-gnu/libpython2.7.so

**Comment by wbrenna** [comment]
I'll check about the manual links. I have never used the python bindings, actually! But you're correct about PageRef being changed to a pointer; I can push that patch in.

**Comment by wbrenna** [comment]
I'm pretty sure your manual linking problem is just because the configure.ac file is old and perhaps was never working correctly for the Python compile. It looks like the autotools aren't finding Python at all on your system; the version and Python commands are just returning empty strings.

**Comment by Corna** [comment]
It should be fixed in Corna/xournalpp@02dc9bae91a56c1152b942a3adda07e1580afbcf
Do you know which is the minimum python version? I've set python >= 2.4, but I don't know the real required version

**Comment by wbrenna** [comment]
Sounds good, I can patch this in. I've never used the python tie-in so I have no idea what version requirements it has; we can fix that easily if it causes bugs for someone.
